esx_service/cli/vmdkops_admin.py

```diff
@@ -1034,5 +1106,223 @@ def tenant_access_ls(args):
+                os.remove(link_path)
+                auth.mode = auth_data.DBMode.NotConfigured
+            info = auth.get_info()
+        except auth_data.DbAccessError as ex:
+            return err_out("Fatal: DB link or DB is corrupted. Check {} ({}".format(link_path, ex))
+
+    if auth.mode  == auth_data.DBMode.NotConfigured:
+        pass
+    elif auth.mode  == auth_data.DBMode.MultiEsx:
+        if not args.force:
+            return err_override("Config DB is already initialized.", info)
+        else:
+            os.remove(link_path)
+    elif auth.mode  == auth_data.DBMode.SingleNode:
+        if not args.force:
+            return err_override("Local DB already exists {}".format(link_path), info)
+        db_move_to_backup(link_path)
+    else:
+        raise Exception("Fatal: Internal error - unknown mode (value %s)" % auth.mode )
+
+
+    if not os.path.exists(db_path):
+        print("Creating new DB at {}".format(db_path))
+        auth = auth_data.AuthorizationDataManager(db_path)
+        err = auth.new_db()
+        if err:
+            return err_out("Init failed: %s" % str(err))
+
+    # Almost done -  just create link and refresh the service
+    if not args.local:
+        print("Creating a symlink to DB at {}".format(link_path))
+        create_db_symlink(db_path, link_path)
+    service_restart()
+    return None
+
+
+def config_rm(args):
+    """
+    Remove Config DB.BaseException
+    :return: None for success, string for error
+    """
+
+    # TODO
+    # This command should ask for double confirmation, reset the DB_PATH,
+    # and delete the database instance
+    # Other ESXs will detect this change (DB is missing) and
+    # reset the DB_PATH (if new DB cannot be discovered)
+
+    err = check_ds_local_args(args)
+    if err:
+        return err
+
+    if not args.force:
+        return err_out("Warning: Config DB removal requires '--force' flag to execute the request")
+
+    if args.datastore:
+        db_path = auth_data.AuthorizationDataManager.ds_to_db_path(args.datastore)
+
+    link_path = auth_data.AUTH_DB_PATH # where was the DB, now is a link
+
+
+    try:
+        os.remove(link_path)
+    except Exception as ex:
+        return err_out("Configuration rm failed ({})".format(ex))
+    print("Removed {}".format(link_path))
+
+    if args.local:
+        return None
+
+    if args.no_backup:
+        os.remove(db_path)
+    else:
+        db_move_to_backup(db_path)
+
+    service_restart()
+    return None
+
+
+def config_mv(args):
+    """
+    Relocate config DB from its current location [not supported yet]
+    :return: None for success, string for error
+    """
+
+    if not args.force:
+        return err_out("Config DB Move to {} ".format(args.to) +
+                       "requires '--force' flag to execute the request.")
+
+    # TODO: question
+    # Should it work only in MultiEsx mode (link and DB exist), should fail in all other modes ?
```

Comment:
agreed - but out of scope for now, I do not think it is important to support all that. We can just ask them to move the file